tests/utils/test_email.py

```diff
@@ -239,3 +240,32 @@ def test_successful_registration_email(mock_smtp):
             email_msg
         )
     destroy_ctfd(app)
+
+
+def test_email_whitelist():
+    app = create_ctfd()
+    with app.app_context():
+        set_config("domain_whitelist", "example.com, uni.acme.com, *.edu, *.edu.de")
```

Comment:
Can you add `*.example.com` and update the test to make sure that we test those cases?

Reply:
I've added some more isolated test cases